### PR TITLE
Update tokens.json

### PR DESCRIPTION
**Pull request by niche500**
## How to add token?

- [x] Add metadata to [`tokens.json`](tokens.json)

```json
    {
    "name": "NOAM",
    "logo": "http://noam.finance/img/noam_logo.png",
    "logo_lg": "http://noam.finance/img/noam_logo.png",
    "symbol": "NOAM",
    "account": "NoamToken",
    "chain": "eos"
    },

```

- [x] Add token logo `*.png` to [`./logos`](./logos) folder
- [x] Update `README.md` with **Symbol** & **Contract Name**

### CODE REVIEW

**Comment by niche500** [comment]
Add files via upload

**Review by niche500 — COMMENTED** [review]
How to add files via upload

**Comment by niche500** [comment]
noam_logo

**Comment by DenisCarriere** [comment]
image

You must add your logo file to the repo and the `tokens.json` url must reference this local repo

**Review by DenisCarriere — CHANGES_REQUESTED** [review]
- [ ] add logo file
- [ ] update `tokens.json` to local repo

**Review by niche500 — COMMENTED** [review]
added the file

File: tokens.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "NOAM",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/noam_logo.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/noam_logo.png",
+        "symbol": "NOAM",
+        "account": "NoamToken",
```
Comment (DenisCarriere) [inline tokens.json:7]: @niche500 this is not a valid EOS account name

Please provide a valid token contract name

**Review by DenisCarriere — CHANGES_REQUESTED** [review]

**Comment by DenisCarriere** [comment]
@niche500 You can re-submit once you have a valid token contract name

**Pull request closed without merging**